From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Herman van Hazendonk <github.com@herrie.org>
Date: Sat, 13 Sep 2026 15:10:00 +0200
Subject: [PATCH] sysbus: grant powermgr the db8 and alarm ACGs

The outbound permission added in the companion patch lets powermgr reach
db8 and the alarm service; it still needs the groups that protect the
methods it calls:

  LunaPmsRoot.cpp:641,673  com.palm.db/putKind, /put
                             -> database.operation
  LunaPmsRoot.cpp:790,859  com.webos.service.alarm/clear, /set
                             -> systemsettings.management (systemservice)

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Pending
---
 files/sysbus/com.webos.service.power2.perm.json | 4 +++-
 1 file changed, 3 insertion(s), 1 deletion(s)

--- a/files/sysbus/com.webos.service.power2.perm.json
+++ b/files/sysbus/com.webos.service.power2.perm.json
@@ -1,4 +1,6 @@
 {
-  "com.webos.service.power2": ["networkconnection.query"
+  "com.webos.service.power2": ["networkconnection.query",
+"database.operation",
+"systemsettings.management"
   ]
 }
